css/dashboard/multiple-choice-answer.less

```diff
@@ -0,0 +1,61 @@
+@import "./helpers";
+
+.multipleChoiceAnswer {
+  width: 100%;
+
+  .icon {
+    text-align: center;
+  }
+
+  .correct {
+    color: #2da343;
+  }
+  .incorrect {
+    color: #ea6d2f;
+  }
+
+  .choice {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    i {
+      flex: 1;
+      line-height: 28px;
+    }
+
+    .choiceContent {
+      flex: 100;
```

Comment:
It seems better to use `width: 100%` in places like this instead of providing an arbitrarily large value to the `flex` ratio.

Reply:
`width: 100%` wouldn't work here. Note that this container has two siblings with `flex: 1`. So the idea is that these two containers take only as much space as they need, and this one takes all the remaining blank space.